packages/core/examples/navbarExample.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import * as React from "react";
+
+import { Navbar, NavbarDivider, NavbarGroup, NavbarHeading, Switch } from "@blueprintjs/core";
+import { BaseExample, handleBooleanChange } from "@blueprintjs/docs";
+
+export interface INavbarExampleState {
+    alignRight?: boolean;
+}
+
+export class NavbarExample extends BaseExample<INavbarExampleState> {
+    public state: INavbarExampleState = {
+        alignRight: false,
+    };
+
+    private handleAlignRightChange = handleBooleanChange(alignRight => this.setState({ alignRight }));
+
+    protected renderExample() {
+        return (
+            <Navbar>
+                <div style={{ margin: "0 auto", width: 480 }}>
+                    <NavbarGroup align={this.state.alignRight ? "right" : "left"}>
+                        <NavbarHeading>Blueprint</NavbarHeading>
```

Comment:
let's move this `NavbarHeading` onto its own `NavbarGroup` for a more realistic example

Reply:
agreed. use two `NavbarGroup`s